### PR TITLE
CSI 4921 support port files comments

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: node/pkg/driver/node_utils.go [hunk 1]
```diff
@@ -17,6 +17,7 @@
 package driver
```
Comment (matancarmeli7) [inline node/pkg/driver/node_utils.go:17]: can you add a unit test for it?

**Review by ELENAGER — APPROVED** [review]
LGTM

File: node/pkg/driver/node_utils.go (hunk 1 above)
Comment (ArbelNathan) [reply node/pkg/driver/node_utils.go:17]: done

**Review by ELENAGER — APPROVED** [review]

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**